Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -825,8 +826,10 @@ class DropDownMenu
     return item;
   }
 
-  private static OnItemClick (this: HTMLAnchorElement): boolean
+  private static OnItemClick (this: HTMLAnchorElement, ev:HTMLElementEventMap["click"]): boolean
```

Comment:
We should use `MouseEvent` instead:

```suggestion
  private static OnItemClick (this: HTMLAnchorElement, ev: MouseEvent): boolean
```

Did we extract the bugfix here or is this still TODO?

Reply:
Extracted